src/platform/user/profile/vet360/components/base/VAPEditView.jsx

```diff
@@ -102,25 +102,29 @@ class VAPEditView extends Component {
       (isFailedTransaction(transaction) ? {} : null);
 
     const actionButtons = (
-      <Vet360EditModalActionButtons
+      <VAPEditModalActionButtons
```

Comment:
Thankfully we don't have any legacy tests that check to see that this `VAPEditView` component renders a `Vet360EditModalActionButtons` child :)